test/integrationTests/implementationProvider.test.ts

```diff
@@ -0,0 +1,33 @@
+/*--------------------------------------------------------------------------------------------- 
+*  Copyright (c) Microsoft Corporation. All rights reserved. 
+*  Licensed under the MIT License. See License.txt in the project root for license information. 
+*--------------------------------------------------------------------------------------------*/
+
+import * as vscode from "vscode";
+import CSharpImplementationProvider from "../../src/features/implementationProvider";
+import CSharpExtensionExports from "../../src/CSharpExtensionExports";
+import * as path from "path";
+import testAssetWorkspace from "./testAssets/testAssetWorkspace";
+import { expect } from "chai";
+
+suite(CSharpImplementationProvider.name, () => {
+    let fileUri: vscode.Uri;
+    
+    suiteSetup(async () => {
+        let csharpExtension = vscode.extensions.getExtension<CSharpExtensionExports>("ms-vscode.csharp");
```

Comment:
I think all our tests share this boilerplate. Can we find a way to share it?

Reply:
@DustinCampbell has already added the helper methods for the integration tests
https://github.com/OmniSharp/omnisharp-vscode/pull/2377. Once that change is in, I can modify the tests here to use that.